quinn-proto/src/frame.rs

```diff
@@ -515,8 +517,7 @@ impl Crypto {
 }
 
 pub struct Iter {
-    // TODO: ditch io::Cursor after bytes 0.5
```

Comment:
You removed this TODO without resolving it -- wondering why?

Reply:
Oh, I remember now. I originally tried resolving that todo by using ``BytesMut``, but ran into some issues with the tests failing. So, I reverted it back to using a cursor while forgetting to put the TODO back. I'll update this in the next push

Reply:
Okay, it's been reverted. Originally, I tried solving the problem, but the tests failed so I reverted everything except that TODO. How might the new TODO be solved? I could include that in this PR if you want.